nix/sources.json

```diff
@@ -101,10 +101,10 @@
         "homepage": "https://pytorch.org/",
         "owner": "stites",
         "repo": "pytorch-world",
-        "rev": "e40fc2133b169b72eb8d772f98b1484dc1d3903f",
+        "rev": "e08419a2f2b9ca177903c9595858b0dc386fc4dd",
```

Comment:
@Cmdv what commit is this? are you sure this is a newer version?

Reply:
There is no need to change, but both are almost the same.

Reply:
I literally did `niv update` and let niv change all the revs

Reply:
ok cool, thanks :)